platform/core/src/extensions/ExtensionManager.ts

```diff
@@ -330,10 +345,6 @@ export default class ExtensionManager {
     return this.dataSourceMap[this.activeDataSource];
   };
 
```

Comment:
why have removed the getDataSource?

Reply:
It is a duplicate of `getActiveDataSource`